spec/opsgenie_alert_tagging_spec.rb

```diff
@@ -133,4 +248,154 @@ def fake_alert_wakinghours_weekend
+      "acknowledged"=>false,
+      "isSeen"=>true,
+      "tags"=>["sleepinghours", "OOH"],
+      "snoozed"=>false,
+      "count"=>1,
+      "lastOccurredAt"=>"2021-08-22T04:21:44.84Z",
+      "createdAt"=>"2021-08-22T04:21:44.84Z",
+      "updatedAt"=>"2021-09-02T12:08:51.21Z",
+      "source"=>"UpdownIO",
+      "owner"=>"",
+      "priority"=>"P3",
+      "teams"=>[{"id"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}],
+      "responders"=>[{"type"=>"team", "id"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}],
+      "integration"=>{"id"=>"ae438e99-b745-452d-a8a3-14b766e0c3fd", "name"=>"Working Hours - updown.io", "type"=>"UpdownIO"},
+      "report"=>{"ackTime"=>278158935, "closeTime"=>278158935, "closedBy"=>"bob@dxw.com"},
+      "ownerTeamId"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}
+  end
+
+  def fake_alert_wakinghours_weekend_tags_existing
+    {"seen"=>true,
+      "id"=>"f4747aac-fa97-4838-8e2f-69992de30a1c-1629601435401",
+      "tinyId"=>"3252",
+      "alias"=>"f4747aac-fa97-4838-8e2f-69992de30a1c-1629601435401",
+      "message"=>"[Updown.io] Timeout was reached",
+      "status"=>"closed",
+      "acknowledged"=>false,
+      "isSeen"=>true,
+      "tags"=>["wakinghours", "OOH"],
+      "snoozed"=>false,
+      "count"=>1,
+      "lastOccurredAt"=>"2021-08-22T13:03:55.401Z",
+      "createdAt"=>"2021-08-22T13:20:55.401Z",
+      "updatedAt"=>"2021-09-02T14:21:51.439Z",
+      "source"=>"UpdownIO",
+      "owner"=>"",
+      "priority"=>"P3",
+      "teams"=>[{"id"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}],
+      "responders"=>[{"type"=>"team", "id"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}],
+      "integration"=>{"id"=>"ae438e99-b745-452d-a8a3-14b766e0c3fd", "name"=>"Working Hours - updown.io", "type"=>"UpdownIO"},
+      "report"=>{"ackTime"=>282828423, "closeTime"=>282828423, "closedBy"=>"bob@dxw.com"},
+      "ownerTeamId"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}
+  end
+
+  def fake_alert_sleepinghours_set_time
+    {"seen"=>true,
+      "id"=>"2052428d-b82a-4198-924f-7b66d568d2e1-1629606104840",
+      "tinyId"=>"3253",
+      "alias"=>"2052428d-b82a-4198-924f-7b66d568d2e1-1629606104840",
+      "message"=>"[Updown.io] Timeout was reached",
+      "status"=>"closed",
+      "acknowledged"=>false,
+      "isSeen"=>true,
+      "tags"=>[],
+      "snoozed"=>false,
+      "count"=>1,
+      "lastOccurredAt"=>"2021-08-22T04:21:44.84Z",
+      "createdAt"=>"2021-08-22T23:00:00.00Z",
+      "updatedAt"=>"2021-09-02T12:08:51.21Z",
+      "source"=>"UpdownIO",
+      "owner"=>"",
+      "priority"=>"P3",
+      "teams"=>[{"id"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}],
+      "responders"=>[{"type"=>"team", "id"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}],
+      "integration"=>{"id"=>"ae438e99-b745-452d-a8a3-14b766e0c3fd", "name"=>"Working Hours - updown.io", "type"=>"UpdownIO"},
+      "report"=>{"ackTime"=>278158935, "closeTime"=>278158935, "closedBy"=>"bob@dxw.com"},
+      "ownerTeamId"=>"72c99d92-c7c8-4fd9-b8db-9cc293cd19e7"}
```

Comment:
Do we need all of this data in our fake? What fields do we actually do anything with? Can we focus on only faking them to keep the amount of space these take up down? Also, watch out that this is real data. We probably shouldn't be committing that. Better to invent new data that matches what we get from the API.